src/viam/sdk/CMakeLists.txt

```diff
@@ -79,18 +79,14 @@ target_sources(viamsdk
     robot/service.cpp
     rpc/dial.cpp
     rpc/server.cpp
+    services/mlmodel/client.cpp
+    services/mlmodel/mlmodel.cpp
+    services/mlmodel/private/proto.cpp
+    services/mlmodel/server.cpp
     services/service.cpp
     spatialmath/geometry.cpp
     spatialmath/orientation_types.cpp
     spatialmath/orientation.cpp
-    tests/test_utils.cpp
```

Comment:
Mocks and tests didn't belong in the SDK library. I've moved them to their own library.